_includes/tutorials/anomaly-detection/ksql/markup/dev/make-src-file.adoc

```diff
@@ -0,0 +1,5 @@
+Now that you have a series of statements that's doing the right thing, the last step is to put them into a file so that they can be used outside the CLI session. Create a file at `src/statements.sql` with the following content:
```

Comment:
Would it make sense to modify as:

```suggestion
Now that you have a series of statements that's doing the right thing, the last step is to put them into a file so that they can be used outside the CLI session. Create a file at `src/statements.sql` with the following content that represents the suspicious names (In production, you would likely use Kafka Connect to read the suspicious names from a database into a Kafka topic, and then create a ksqlDB stream for it).
```